android/brave_java_resources.gni

```diff
@@ -313,6 +313,7 @@ brave_java_resources = [
   "java/res/drawable-xxxhdpi/uphold_green.png",
   "java/res/drawable-xxxhdpi/uphold_white.png",
   "java/res/drawable-xxxhdpi/yandex.png",
+  "java/res/drawable/above_shadow.xml",
```

Comment:
putting these all together in a single list and directory seems totally unmanageable. How do know which resources go with what? There is a major problem here in general, but for this PR we need to split these out into their own `android_resources` under playlist

Reply:
Fixed in https://github.com/brave/brave-core/pull/26628/commits/3e2b46d24e3bf5ea1a9efa58fc0e63e939703c71